backend/db/manage_db.py

```diff
@@ -1 +1,83 @@
-# TODO create the database for the app
+from flask import Flask
+from sqlalchemy import create_engine  
+from sqlalchemy import Column, String, Float, Boolean
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.declarative import declarative_base  
+from sqlalchemy.orm import sessionmaker
+
+from flask_script import Manager
+from flask_migrate import Migrate, MigrateCommand
+
+from flask_sqlalchemy import SQLAlchemy
+
+import psycopg2
+
+app = Flask(__name__)
+
+db_string = "postgres://xpcdulaqqancpa:0616d8130a659a869dacb11768aaf0d8f87f741b19fc8bf46d3af34c78f5ca85@ec2-34-200-72-77.compute-1.amazonaws.com:5432/ddjr78vh1cvcc4"
```

Comment:
It's generally bad practice to leave the DB password in source code, so I created #22 to fix that. You are also welcome to implement a fix on your own if you disagree with how I did it